packages/preview/showybox-1.0.0/typst.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "showybox"
+version = "1.0.0"
+entrypoint = "showy.typ"
+authors = ["Pablo González Calderón", "Showybox Contributors"]
+license = "MIT"
+description = "Colorful and customizable boxes for typst"
```

Comment:
please add `exclude = ["assets", "Showybox's Manual.pdf"]` to drastically decrease the package size for users

Reply:
Done ;)